fvm/evm/stdlib/contract.cdc

```diff
@@ -102,14 +109,20 @@ contract EVM {
 
         /// The EVM address of the cadence owned account
         access(all)
-        fun address(): EVMAddress {
+        view fun address(): EVMAddress {
             // Always create a new EVMAddress instance
             return EVMAddress(bytes: self.addressBytes)
         }
 
+        /// The EVM address of the cadence owned account behind an entitlement, acting as proof of access
+        access(Validate)
+        view protectedAddress(): EVMAddress {
+            return self.address()
+        }
```

Comment:
What is this for?

Reply:
My thinking here is that it serves as proof that the owner of this reference does in fact own this COA. Consider something like

```cadence
access(all) fun claimAirdrop(coa: auth(EVM.Validate) &EVM.CadenceOwnedAccount): @ExampleToken.Vault? {
  if let dropAmount = self.allowlist[coa.address()] {
    return <- self.mintTokens(amount: dropAmount)
  }
  return nil
}
```

Without something like the above, how would one prove ownership of a COA without also exposing sensitive methods like `call`, `deploy`, etc or necessitating a signature?

Reply:
I see 👍